src/web/mjs/engine/Connector.mjs

```diff
@@ -55,6 +57,7 @@ export default class Connector {
      */
     async _initializeConnector() {
         let uri = new URL(this.url);
+        this.retries = parseInt(Engine.Settings.retries.value) || this.retries;
```

Comment:
this should really just be in the fetches (fetchDOM, fetchJSON, etc)
as it is right now you can't change the setting after the connector initialize

Reply:
I second this, retrying should be the exception and not the rule. I suggest to point out what connectors have problems, analyze the cause and just add the retry to where it is acceptable.